unittests/test-contracts/test_ram_limit/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 if( SYSIO_COMPILE_TEST_CONTRACTS )
+   # add_contract( test_ram_limit test_ram_limit test_ram_limit.cpp )
```

Comment:
Remove the commented out code which is the only change.

Reply:
this was intentionally left in to show how to build manually (as it needs that additional target_compile_options config)